packages/contracts-bedrock/contracts/L2/GasPriceOracle.sol

```diff
@@ -129,6 +136,15 @@ contract GasPriceOracle is Semver {
         return L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).basefee();
     }
 
+    /**
+     * @notice Retrieves the number of decimals used in the scalar.
```

Comment:
This should have `@custom:legacy`

Reply:
The others don't have this but happy to add in another PR